### PR TITLE
Changed the permissions on the Jar File to execute only the executabl…

### PR DESCRIPTION
**Pull request by pats-john**
…e Jar File

**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,22 +1,8 @@
...
-#FROM hmctspublic.azurecr.io/base/java:openjdk-11-distroless-1.2
-
-#COPY lib/AI-Agent.xml /opt/app/
-#COPY build/libs/spring-boot-template.jar /opt/app/
-
-#EXPOSE 4550
-#CMD [ "spring-boot-template.jar" ]
-
 FROM openjdk:11
 MAINTAINER HMI-QA
```
Comment (karanotts) [inline Dockerfile:2]: https://docs.docker.com/engine/reference/builder/#maintainer-deprecated

File: Dockerfile [hunk 2]
```diff
@@ -1,22 +1,8 @@
...
 WORKDIR /opt/app/wiremock
 COPY ./wiremock/ /opt/app/wiremock/
-RUN chmod 777 *
+RUN chmod a+x wiremock-jre8-standalone-2.27.1.jar
 RUN ls -al /opt/app/wiremock/
-
 EXPOSE 8080:8080
-#RUN java -jar wiremock-jre8-standalone-2.27.0.jar
-ENTRYPOINT ["java", "-jar", "wiremock-jre8-standalone-2.27.1.jar"]
+ENTRYPOINT ["java", "-jar", "wiremock-jre8-standalone-2.27.1.jar"]
```
Comment (karanotts) [inline Dockerfile:8]: Missing newline, please be mindful of linting

Comment (pats-john) [reply Dockerfile]: Added a new line at the end of the File.....

**Comment by pats-john** [comment]
Merge to master completed and branch deleted.

**Pull request merged**